partcad/src/partcad/actions/part_actions.py

```
@@ -1,11 +1,13 @@
-import os
-import shutil
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)